projects/1.19/assets/dimensional-pockets-ii/dimensionalpocketsii/lang/zh_cn.json

```
@@ -0,0 +1,1072 @@
+	"dimensionalpocketsii.gui.config.use_items_info": "是否允许物品能在口袋内使用",
+	"dimensionalpocketsii.gui.config.use_items_info_two": "参见下方关于当前已被限制的物品的列表",
+	
+	"dimensionalpocketsii.gui.config.use_commands": "能够使用命令",
+	"dimensionalpocketsii.gui.config.use_commands_info": "是否允许命令能在口袋内使用",
+	"dimensionalpocketsii.gui.config.use_commands_info_two": "参见下方关于当前已被限制的命令的列表",
+	
+	"dimensionalpocketsii.gui.config.chunks": "区块加载",
+	"dimensionalpocketsii.gui.config.chunks_info": "是否允许口袋次元内的区块保持加载",
+	
+	"dimensionalpocketsii.gui.config.walls": "破坏围墙",
+	"dimensionalpocketsii.gui.config.walls_info": "是否允许在创造模式下破坏口袋内的围墙",
+	
+	"dimensionalpocketsii.gui.config.book": "携带次元手册出生",
+	"dimensionalpocketsii.gui.config.book_info": "是否允许每一位初次进入世界的玩家都获得一本次元手册",
+	
+	"dimensionalpocketsii.gui.config.replace": "更换部件",
+	"dimensionalpocketsii.gui.config.replace_info": "不论是否降低了口袋内的高度，如果选项过大，其高度则会越小",
+	
+	"dimensionalpocketsii.gui.config.hostile": "停止敌对生物生成",
+	"dimensionalpocketsii.gui.config.hostile_info": "是否允许敌对生物在口袋内停止生成",
+	
+	"dimensionalpocketsii.gui.config.message.info": "内容信息",
+	"dimensionalpocketsii.gui.config.message.info_desc": "是否允许本模组在控制台或日志内输出内容信息[INFO]",
+	
+	"dimensionalpocketsii.gui.config.message.debug": "调试信息",
+	"dimensionalpocketsii.gui.config.message.debug_desc": "是否允许本模组在控制台或日志内输出调试信息[DEBUG]",
+	
+	"dimensionalpocketsii.gui.config.textures": "连接纹理",
+	"dimensionalpocketsii.gui.config.textures_info": "是否允许口袋内的连接纹理能正常工作",
+	
+	"dimensionalpocketsii.gui.config.blocked_structures": "被限制的结构",
+	"dimensionalpocketsii.gui.config.blocked_structures_info": "关于口袋内被限制的结构的列表",
+	
+	"dimensionalpocketsii.gui.config.blocked_items": "被限制的物品",
+	"dimensionalpocketsii.gui.config.blocked_items_info": "关于口袋内被限制的物品的列表",
+	
+	"dimensionalpocketsii.gui.config.op_level": "OP命令等级",
+	"dimensionalpocketsii.gui.config.op_level_slide": "OP 等级：",
+	"dimensionalpocketsii.gui.config.op_level_info": "允许你能更改绕开命令限制所需要的服务器管理员（OP）等级",
+	"dimensionalpocketsii.gui.config.op_level_info_two": "数字范围限制在0 到 4 之间",
+	
+	"dimensionalpocketsii.gui.config.blocked_commands": "被限制的命令",
+	"dimensionalpocketsii.gui.config.blocked_commands_info": "关于口袋内被限制的命令的列表",
+	
+	"dimensionalpocketsii.gui.config.backups": "创建备份",
+	"dimensionalpocketsii.gui.config.backups_info": "是否允许本模组创建包含口袋内信息的备份文件",
+	
+	"dimensionalpocketsii.gui.config.add": "增加元素",
+	"dimensionalpocketsii.gui.config.remove": "移除元素",
+	
+	"dimensionalpocketsii.gui.config.message.restart": "此选项需要重新启动游戏后才能生效",
+	
+	"dimensionalpocketsii.gui.done": "完成",
+	
+	"dimensionalpocketsii.gui.help.crafter.crafting_slots": "合成栏",
+	"dimensionalpocketsii.gui.help.crafter.crafting_slots_one": "就像一个普通的工作台一样",
+	"dimensionalpocketsii.gui.help.crafter.crafting_slots_two": "你能使用这个部件来合成物品",
+	
+	"dimensionalpocketsii.gui.help.crafter.result_slot": "输出栏",
+	"dimensionalpocketsii.gui.help.crafter.result_slot_one": "这里是被合成出的物品输出的地方，",
+	"dimensionalpocketsii.gui.help.crafter.result_slot_two": "当你有一个有效的合成配方的时候",
```

```suggestion
	"dimensionalpocketsii.gui.help.crafter.result_slot_one": "当存在有效的合成配方时",
	"dimensionalpocketsii.gui.help.crafter.result_slot_two": "合成出的物品会出现在此",
```
原文应该是一段话拆成了几部分
类似句式请改成适合中文的版本